alpha/external-web/app/routes/about-you.js

```diff
@@ -87,3 +99,20 @@ function updateEligibilityFlag (id, newID) {
       })
   }
 }
+
+function validationErrors (request, response, next) {
```

Comment:
split out into validator-about-you.js which is included as a require and called in post